packages/opentelemetry-exporter-jaeger/src/transform.ts

```diff
@@ -71,11 +75,22 @@ export function spanToThrift(span: ReadableSpan): ThriftSpan {
     })
   );
 
+  if (span.instrumentationLibrary) {
```

Comment:
Should it also check if name and version are set before adding? 
Spec doesn't state this scenario

Reply:
I don't think so because name is mandatory and version is optional as described [here](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#get-a-tracer), If the user doesn't specify version then it will be undefined.